.devops/repo/README.md

```diff
@@ -0,0 +1,47 @@
+## Instructions
```

Comment:
Perhaps there should be a note here that there are several assumptions here, in case someone wants to run this locally? (rust toolchain installed, forc installed, etc.)

Reply:
Good point. This is meant for internal use so the assumption was that people have what we use but yh, I'll add some things